RevenueCatUI/CustomerCenter/Views/ManageSubscriptionsView.swift

```diff
@@ -142,20 +142,51 @@ struct SubscriptionDetailsView: View {
 
     var body: some View {
         VStack(alignment: .leading, spacing: 8) {
-            Text("\(subscriptionInformation.title) - \(subscriptionInformation.durationTitle)")
-                .font(.subheadline)
-                .padding([.horizontal, .top])
-
-            Text("\(subscriptionInformation.price)")
-                .font(.caption)
-                .foregroundColor(Color.gray)
-                .padding(.horizontal)
-
-            if let nextRenewal =  subscriptionInformation.nextRenewalString {
-                Text("\(subscriptionInformation.renewalString): \(String(describing: nextRenewal))")
+            VStack(alignment: .leading) {
+                Text("\(subscriptionInformation.title)")
+                    .font(.headline)
+                Text("\(subscriptionInformation.explanation)")
+                    .frame(maxWidth: 200, alignment: .leading)
                     .font(.caption)
                     .foregroundColor(Color.gray)
-                    .padding([.horizontal, .bottom])
+            }.padding([.bottom], 10)
+
+            HStack(alignment: .center) {
+                Image(systemName: "coloncurrencysign.arrow.circlepath")
+                    .frame(width: 22)
+                VStack(alignment: .leading) {
+                    Text("Billing cycle")
```

Comment:
I guess we will need to extract all these strings so they are provided by the backend and used here, but that can come in a separate PR.

Reply:
As discussed on Slack, this will be done in a separate PR.